newrelic-agent/src/test/java/com/newrelic/agent/transport/DataSenderImplTest.java

```diff
@@ -50,10 +51,7 @@
 import java.net.URL;
 import java.security.NoSuchAlgorithmException;
 import java.text.MessageFormat;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Make sure that you're using the java-agent code style found at `newrelic-java-agent/dev-tools/code-style/java-agent-code-style.xml`. It's based on the [Google Style Guide](https://google.github.io/styleguide/javaguide.html) which doesn't allow wildcard imports. You can import the scheme in **Preferences > Editor > Code Style > Java > Scheme**.

Reply:
Done